codec-http/src/test/java/io/netty/handler/codec/http/cookie/ClientCookieDecoderTest.java

```diff
@@ -25,14 +25,16 @@
 import java.util.Iterator;
 import java.util.TimeZone;
 
+import static org.hamcrest.CoreMatchers.instanceOf;
+import static org.hamcrest.CoreMatchers.is;
 import static org.junit.Assert.*;
 
 public class ClientCookieDecoderTest {
     @Test
     public void testDecodingSingleCookieV0() {
         String cookieString = "myCookie=myValue;expires="
                 + DateFormatter.format(new Date(System.currentTimeMillis() + 50000))
-                + ";path=/apathsomewhere;domain=.adomainsomewhere;secure;";
+                + ";path=/apathsomewhere;domain=.adomainsomewhere;secure;SameSite=None";
```

Comment:
See above, I don't think None is a possible value over the wire, only in the storage model.

Reply:
See my response above about the actual implementation in browsers.